### ISSUE TITLE
Redis.RENAME not working correctly with RedisTsdb.LABELS

### ISSUE THREAD

**Issue by leeaaron629** [RedisTimeSeries/RedisTimeSeries#768]
Hi,

This is in RedisTSDB 1.4.7:

```
redis-ts.dev.local:6379> ts.create prices10 
OK
redis-ts.dev.local:6379> ts.alter prices10 LABELS id 10 frequency 60
OK
// Key created with name prices10 and ^ LABELS
redis-ts.dev.local:6379> ts.queryindex id=10
1) "prices10" <-- queryindex is okay
redis-ts.dev.local:6379> rename prices10 prices1
OK
redis-ts.dev.local:6379> ts.queryindex id=10
1) "prices10" <-- queryindex return old priceName=prices10
redis-ts.dev.local:6379> ts.info prices10
(error) ERR TSDB: the key does not exist <-- Key does not exists
redis-ts.dev.local:6379> ts.info prices1 <-- Key does exists
 1) totalSamples
 2) (integer) 0
 ```
The above scenario is the same with TS.MGET or TS.MRANGE:

```
redis-ts.dev.local:6379> ts.create prices20 LABELS id 20 type TEST
OK
redis-ts.dev.local:6379> ts.mget FILTER id=20
1) 1) "prices20"
   2) (empty array)
   3) (empty array)
redis-ts.dev.local:6379> RENAME prices20 prices2
OK
redis-ts.dev.local:6379> ts.mget FILTER id=20
(empty array)
```

I am using RENAME to delete a key -> time-series without any downtime atm. Maybe there's a better way to accomplish this?

**Comment by gkorland** [comment]
@leeaaron629 please notice support for rename was only added at 1.4.9.
```
127.0.0.1:6379> ts.create prices10
OK
127.0.0.1:6379>  ts.alter prices10 LABELS id 10 frequency 60
OK
127.0.0.1:6379> ts.queryindex id=10
1) "prices10"
127.0.0.1:6379> rename prices10 prices1
OK
127.0.0.1:6379> ts.queryindex id=10
1) "prices1"
```

**Comment by leeaaron629** [comment]
Perfect, thanks for letting me know. I will do an upgrade to 1.4.9 then.

**Comment by leeaaron629** [comment]
Was able to fix it by upgrading to 1.4.9. Thanks!
